src/domain/entities/figma-team.ts

```diff
@@ -44,6 +44,14 @@ export class FigmaTeam {
 	get adminInfo() {
 		return this.#adminInfo;
 	}
+
+	toFigmaTeamSummary(): FigmaTeamSummary {
```

Comment:
**note:** Had to include this change to resolve conflicts with recent changes from Roy (https://github.com/atlassian-labs/figma-for-jira/pull/118/files#diff-473697600a010a6f00bb27d2fa9a0c1d0bcd7bfa25071d3b203318cb8c6d7c2fR43).